receiver/otlpreceiver/otlp.go

```diff
@@ -141,18 +141,6 @@ func (r *otlpReceiver) startProtocolServers(host component.Host) error {
 		if err != nil {
 			return err
 		}
-		if r.cfg.GRPC.NetAddr.Endpoint == defaultGRPCEndpoint {
-			r.logger.Info("Setting up a second GRPC listener on legacy endpoint " + legacyGRPCEndpoint)
-
-			// Copy the config.
-			cfgLegacyGRPC := r.cfg.GRPC
-			// And use the legacy endpoint.
-			cfgLegacyGRPC.NetAddr.Endpoint = legacyGRPCEndpoint
-			err = r.startGRPCServer(cfgLegacyGRPC, host)
-			if err != nil {
-				return err
-			}
-		}
```

Comment:
This needs to wait for a bit until we can remove. Probably one two releases at least.

Reply:
Sure, should we target the deprecation in v0.29.0? I can set a reminder after v0.28.0 is out.

Reply:
Maybe 0.30.0 to give some more time to the users?

Reply:
Waiting for you to remove this from the PR to be able to merge it.

Reply:
Friendly ping @mxiamxia  :)

Reply:
Done!! Excuse me with the late response. :)

Reply:
> Maybe 0.30.0 to give some more time to the users?

Sure. I am fine with it as well. Ty!